src/EditorFeatures/CSharpTest/PullMemberUp/CSharpPullMemberUpTests.cs

```diff
@@ -17,6 +17,7 @@
 using System.Collections.Generic;
 using Microsoft.CodeAnalysis.Test.Utilities.PullMemberUp;
 using Roslyn.Test.Utilities;
+using System.Xml.Linq;
```

Comment:
Is this using needed?

Reply:
sort.

Reply:
I really wish if we wanted usings sorted it would at least fail the correctness test 

\</rant>